src/components/Header.js

```diff
@@ -0,0 +1,96 @@
+import React, {useEffect, useState} from 'react';
+import {makeStyles} from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import MenuItem from '@material-ui/core/MenuItem';
+import Menu from '@material-ui/core/Menu';
+import Button from '@material-ui/core/Button';
+import {LogoutUser, getToken} from "../store/users/actions"
```

Comment:
Make imports in all files be more readable
example: 
import module from 'external import'
import module1 from 'external import1'

import module from 'internal import'

import styles from 'styles'